src/status_im/data_store/realm/core.cljs

```diff
@@ -49,11 +49,9 @@
   ;; TODO: remove for release 0.9.18
   ;; delete the realm file if its schema version is higher
   ;; than existing schema version (this means the previous
-  ;; install has incompatible database schemas)
-  (if (> (realm-version file-name)
-         (apply max (map :schemaVersion base/schemas)))
-    (reset-realm file-name schemas)
-    (migrate-realm file-name schemas)))
+  ;; install has incompatible database schemas) 
+  (reset-realm file-name schemas)
```

Comment:
have you tested this ? To me it seems like it will just delete the db everytime you start the app ?

Reply:
You are correct